src/index.js

```diff
@@ -1,14 +1,107 @@
 import _ from 'lodash';
 import './style.css';
+import refreshImage from './refresh.png';
+import arrow from './arrow-key.svg';
+import dots from './dots.svg';
+import trash from './trash.svg';
 
- function component() {
-   const element = document.createElement('div');
+// selecting the container
+const container = document.querySelector('.container');
 
-  // Lodash, now imported by this script
-   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-   element.classList.add('hello');
+// selecting the header section
+const headerSection = document.querySelector('.header');
 
-   return element;
- }
+// making the refresh button image
+const refresh = document.createElement('img');
+refresh.classList.add('refreshButton');
+refresh.src = refreshImage;
 
- document.body.appendChild(component());
\ No newline at end of file
+headerSection.append(refresh);
+
+// selecting the input container
+const inputSection = document.querySelector('.inputSection');
+
+// selecting the input field
+const inputField = document.getElementById('inputField');
+
+const arrowHolder = document.createElement('img');
+arrowHolder.classList.add('arrow');
+arrowHolder.src = arrow;
+
+inputSection.append(inputField, arrowHolder);
+
+// selecting the button
+const button = document.querySelector('.clearAll');
+
+// implementing list functionality
+
+const taskHolder = [];
```

Comment:
- [ ] Based on the project requirement, it is expected that on page load, you render dynamically, the created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task. I suggest you use Windows `load` events which fire when the **_whole page_** has loaded, including all dependent resources such as stylesheets, scripts, iframes, and images.

```javascript
  window.addEventListener("load", renderTasks);
```

The `renderTasks` function in the code snippet above is the same as your `addToDom` function. 

Kindly implement this.